plugins/Rockets/RocketsPlugin.cpp

```diff
@@ -1569,6 +1583,42 @@ class RocketsPlugin::Impl : public ActionInterface
                       buildJsonSchema(props, hyphenatedToCamelCase(endpoint)));
     }
 
+    void _handleloadersSchema()
+    {
+        const RpcDescription desc{LOADERS_SCHEMA,
+                                  "Get the schema for all loaders"};
+
+        _bindEndpoint(
+            LOADERS_SCHEMA, [&](const rockets::jsonrpc::Request& /*request*/) {
+                std::vector<std::pair<std::string, PropertyMap>> props =
+                    _engine.getScene()
+                        .getLoaderRegistry()
+                        .getLoaderPropertyMaps();
+
+                std::vector<std::string> schemas;
+                for (const auto& kv : props)
+                    schemas.push_back(buildJsonSchema(kv.second, kv.first));
+
+                const size_t numSchemas = schemas.size();
+                if (numSchemas == 0)
+                    return Response{"[]"};
+
+                std::string result = "[\n";
```

Comment:
`to_json` should work to avoid the manual JSON string formatting here. If it doesn't work out of the box, I'd still encourage you to implement to make this less verbose here.

Reply:
Done.